templates/CRM/Custom/Page/Option.tpl

```diff
@@ -157,7 +157,7 @@
       {/literal}
 
       <div class="action-link">
-          {crmButton q="reset=1&action=map&fid=$fid&gid=$gid" class="action-item" icon="sort-alpha-asc"}{ts}Alphabetize Options{/ts}{/crmButton}
+          {crmButton q="reset=1&action=map&fid=$fid&gid=$gid" class="action-item no-open" icon="sort-alpha-asc"}{ts}Alphabetize Options{/ts}{/crmButton}
```

Comment:
There's already a class for this: `no-popup` so I don't think you need to create this new one.

Reply:
@colemanw I tried using that class, but if I click Alphabetize in a popup window it redirects to the page version. This may be a bug with that class but adding in this class which is only excluded in this one line and not in both places was the only way I could get around that bug.